### DISCUSSION TITLE
shadowsocks-windows是否可以向sagernet一样支持丰富的plugin呢？

### DISCUSSION THREAD

**Discussion by xuemind** [shadowsocks/shadowsocks-windows#3311] [General]
shadowsocks-windows是我用过最好用的windows代理，但是仅支持shadowsocks代理，是否可以想sagernet一样支持更多的协议的plugin呢？

**Comment by chenshaoju** [comment]
目前我个人已知的插件清单：https://github.com/shadowsocks/shadowsocks-windows/issues/2228#issuecomment-465047836

**Reply by fernvenue** [reply]
Maybe you can add [teddysun/xray-plugin](https://github.com/teddysun/xray-plugin) to this list also :)
